naga/tests/out/wgsl/access.wgsl

```diff
@@ -121,11 +121,15 @@ fn assign_through_ptr_fn(p: ptr<function, u32>) {
     return;
 }
 
-fn assign_array_through_ptr_fn(foo_2: ptr<function, array<vec4<f32>, 2>>) {
-    (*foo_2) = array<vec4<f32>, 2>(vec4(1f), vec4(2f));
+fn assign_array_through_ptr_fn(foo_3: ptr<function, array<vec4<f32>, 2>>) {
+    (*foo_3) = array<vec4<f32>, 2>(vec4(1f), vec4(2f));
     return;
 }
 
+fn array_by_value(a_1: array<i32, 5>, i: i32) -> i32 {
+    return a_1[i];
+}
+
```

Comment:
question: @gfx-rs/naga, should this be showing up in final WGSL output at all if it's not used? It doesn't seem to be consumed at all in the rest of the shader.

Reply:
Answer: Ah, this is here because it's explicitly defined in the `in` shader, and AFAIK we don't have compaction enabled for these snapshots. Nevermind!